platform/linux-generic/include/odp/api/plat/ipsec_inlines.h

```diff
@@ -19,10 +21,13 @@
 	#define _ODP_INLINE static inline
 	#define odp_ipsec_packet_from_event __odp_ipsec_packet_from_event
 	#define odp_ipsec_packet_to_event __odp_ipsec_packet_to_event
+	#define odp_ipsec_result __odp_ipsec_result
 #else
 	#define _ODP_INLINE
 #endif
 
+extern const _odp_packet_inline_offset_t _odp_packet_inline;
```

Comment:
Shouldn't this be in platform/linux-generic/include/odp/api/plat/packet_inline_types.h, where the get function that uses it is defined?

Reply:
A good point. Added a commit in V2, which moves the declarations in all inline headers.